### PR TITLE
Html controls form

### PR DESCRIPTION
**Pull request by AndiAJ**
• Update versioning to v3.0
• Create new `HTMLforms.html` page

### CODE REVIEW

File: v3.0/htmlControlsForm.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html_Control_Form</title>
```
Comment (AaronMT) [inline v3.0/htmlControlsForm.html]: @AndiAJ I think we should keep `externalLinks.html` as is and just create a new page `HTMLforms.html` `HTML` caps

**Review by AaronMT — CHANGES_REQUESTED** [review]
Let's break this out and keep `externalLinks`

File: v3.0/htmlControlsForm.html (hunk 1 above)
Comment (AndiAJ) [reply v3.0/htmlControlsForm.html]: Sure, thanks !

Comment (AndiAJ) [reply v3.0/htmlControlsForm.html]: Done !

**Comment by AndiAJ** [comment]
Closing, as we decided to not automate these sort of tests.

**Pull request closed without merging**